src/wlapplication.h

```diff
@@ -178,6 +178,9 @@ struct WLApplication {
 	// Handle the given pressed key. Returns true when key was
 	// handled.
 	bool handle_key(bool down, const SDL_Keycode& keycode, int modifiers);
+	void enable_handle_key(bool enable_) {
+		handle_key_enabled = enable_;
```

Comment:
Please use a trailing underscore for member variables and no trailing underscore for local variables. So those should be `handle_key_enabled_` and `enable`.